girder/models/setting.py

```diff
@@ -209,6 +210,18 @@ def validateCoreUploadMinimumChunkSize(self, doc):
             'Upload minimum chunk size must be an integer >= 0.',
             'value')
 
+    def validateCoreFilesystemDefaultFileMode(self, doc):
+        try:
+            doc['value'] = int(doc['value'])
+            # Make sure that mode is still rw for user
+            if (stat.S_IRUSR | stat.S_IWUSR) & doc['value']:
```

Comment:
Or it might be easier to read as

    if not doc['value'] & stat.S_IRUSR or not doc['value'] & stat.S_IWUSR:
        raise ValidationException(...)